frontend/src/App.js

```diff
@@ -1,22 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const [tubeData, setTubeData] = useState([]);
+
+  const lines = "bakerloo,central,circle,district,hammersmith-city,jubilee,metropolitan,northern,piccadilly,victoria,waterloo-city";
+
+  useEffect(() => {
+    axios.get(`https://api.tfl.gov.uk/Line/${lines}/Arrivals?`)
```

Comment:
Could you let us know why you decided to use axios?

Reply:
It seemed like a win-win over fetch. No apparent disadvantages but better error handling and browser compatibility. Easy to implement and you don't have to redirect chat gtp so much apparently.